.ci/targets/android_platform_tests.yaml

```diff
@@ -2,10 +2,10 @@ tasks:
   - name: prepare tool
     script: .ci/scripts/prepare_tool.sh
     infra_step: true # Note infra steps failing prevents "always" from running.
-  - name: download android deps
+  - name: download Dart and android deps
```

Comment:
Seems like the `android` in these names should be capitalized.

Reply:
Fixed.